hazelcast-client/src/test/java/com/hazelcast/client/ClientRegressionWithMockNetworkTest.java

```diff
@@ -835,4 +843,50 @@ public void testMemberAddedWithListeners_thenCheckOperationsNotHanging() {
         String key = generateKeyOwnedBy(h2);
         map.get(key);
     }
+
+
+    @Test
+    @Category(SlowTest.class)
+    public void testServerShouldNotCloseClientWhenClientOnlyListening() {
+        Config config = new Config();
+        int clientHeartbeatSeconds = 8;
```

Comment:
why 8 secs and not a shorter duration? This seems to cause it to be a slow test.

Reply:
When it is shorter it could fail when environment is slow. That is why I wanted to keep it longer.